syntax = "proto3";

package cornserve.common;

// Whether something was successful or not
enum Status {
  STATUS_UNSPECIFIED = 0;
  STATUS_OK = 1;
  STATUS_ERROR = 2;
}